lib/models/authenticator.js

```diff
@@ -304,36 +304,23 @@ class TlsAuthenticator extends Authenticator {
   }
 
   /**
-   * Ensures that the extracted WebID URI is hosted on this server. If it is,
-   * returns a UserAccount instance for that WebID, throws an error otherwise.
+   * Returns a user account instance for a given Web ID.
    *
    * @param webId {string}
    *
-   * @throws {Error} If the account is not hosted on this server
-   *
    * @return {UserAccount}
    */
-  ensureLocalUser (webId) {
+  loadUser (webId) {
     const serverUri = this.accountManager.host.serverUri
 
     if (domainMatches(serverUri, webId)) {
       // This is a locally hosted Web ID
-      return Promise.resolve(this.accountManager.userAccountFrom({ webId }))
-    }
-
-    debug(`WebID URI ${JSON.stringify(webId)} is not a local account, verifying authorized provider`)
-
-    return this.discoverProviderFor(webId)
-      .then(authorizedProvider => {
-        debug(`Authorized provider for ${webId} is ${authorizedProvider}`)
+      return this.accountManager.userAccountFrom({ webId })
+    } else {
+      debug(`WebID URI ${JSON.stringify(webId)} is not a local account, using it as an external WebID`)
```

Comment:
Yes, but are we still checking that username/password authentication for that account is not enabled? Because otherwise, I could register on any Solid pod with anybody's Web ID.

Reply:
You can't register with anybody's Web ID since the server still checks your profile against your browser cert. So if you don't have the cert, the login will still fail

Reply:
The regular webid-tls checking happens earlier in the workflow, in `extractWebId()` in authenticator / line 274

Reply:
Okay, happy that's covered!